src/triage/predictlist/__init__.py

```diff
@@ -288,14 +204,15 @@ class Retrainer:
         model_group_id (string)
     """
     def __init__(self, db_engine, project_path, model_group_id):
+        self.retrain_hash = None
         self.db_engine = db_engine
         upgrade_db(db_engine=self.db_engine)
         self.project_storage = ProjectStorage(project_path)
         self.model_group_id = model_group_id
-        self.model_trainer = None
         self.matrix_storage_engine = self.project_storage.matrix_storage_engine()
-        self.run_id, self.experiment_config = experiment_config_from_model_group_id(self.db_engine, self.model_group_id)
+        self.experiment_run_id, self.experiment_config = experiment_config_from_model_group_id(self.db_engine, self.model_group_id)
         self.training_label_timespan = self.experiment_config['temporal_config']['training_label_timespans'][0]
+        self.test_label_timespan = self.experiment_config['temporal_config']['test_label_timespans'][0]
```

Comment:
I think these need to be determined with respect to the specific model group, not just the first entry in the experiment config (that might not be the one that was actually used for this model group). Likewise for the `max_training_history`